src/platforms/java/guides/spring/advanced-usage.mdx

```diff
@@ -29,6 +31,133 @@ import io.sentry.spring.EnableSentry
 class SentryConfiguration
 ```
 
+2. Register the servlet filter bean `SentryUserFilter`:
```

Comment:
this section could have a `note` which makes it available under 5.0 as a beta version still

Reply:
Not sure how to phrase it specifically to this point. Feel free to create a PR :-)

Reply:
https://docs.sentry.io/contributing/pages/components/#alert

just add an alert before that explaining this is available under SDK v5 and its beta still, we've done that before when SDKs were beta or when features were beta too, pretty sure you can make it :)